### DISCUSSION TITLE
How to modify rootless container's files by using `podman mount`?

### DISCUSSION THREAD

**Discussion by kasraghu** [containers/podman#20549] [Q&A]
Hello,
I am trying to edit files of a rootless container from host (`podman version 4.6.2` on Debian Trixie).

I have run `podman unshare` and `podman mount ContainerID`. The container filesystem is available on host  `~/.local/share/containers/storage/vfs/dir/0123`. I have added  file at  `~/.local/share/containers/storage/vfs/dir/0123` on host but, this file is not available inside the container.

If I do `podman cp test.txt ContainerID:/test`, the file is available on the container. What should I do to use the `mount` option properly?

**Comment by rhatdan** [comment]
First start using overlayfs not vfs for the storage layer.
